gdk/src/keys.rs

```diff
@@ -44,6 +45,18 @@ impl IntoGlib for Key {
 }
 
 impl Key {
+    #[doc(alias = "gdk_keyval_from_name")]
+    pub fn from_name(keyval_name: &str) -> Self {
+        skip_assert_initialized!();
+        Self(unsafe { ffi::gdk_keyval_from_name(keyval_name.to_glib_none().0) })
+    }
+
+    #[doc(alias = "gdk_unicode_to_keyval")]
+    pub fn from_unicode(c: char) -> Self {
```

Comment:
This can't fail?

Reply:
It returns "the corresponding GDK key symbol, if one exists. or, if there is no corresponding symbol, wc | 0x01000000". I'm not really sure how that is supposed to make sense, but I guess it doesn't count as a "failure" and is a valid "keyval"?